TeamcityRestApi/TeamcityConnector.fs

```diff
@@ -825,6 +828,18 @@ type TeamcityConnector(httpconnector : IHttpTeamcityConnector) =
             let payload = sprintf """<builds><build id="%s"/></builds>""" buildid
             let requestDAta = httpconnector.HttpPutXmlContent(conf, url, payload)
             requestDAta.IsSuccessful
+        
+        member this.GetBranchesFromTeamcityConfiguration(conf: ITeamcityConfiguration, buildTypeId: string) =
+            let url = sprintf "app/rest/buildTypes/id:%s/branches" buildTypeId
+            let content = httpconnector.HttpRequest(conf, url, RestSharp.Method.Get).Content
+            let branches = new System.Collections.Generic.List<string>()
```

Comment:
u dont need to use System.Collections.Generic.List<string>, u can use generic FSharp lists, see how other functions are done

Reply:
All other functions also use System.Collections.Generic.List,

Reply:
ah ok, u can keep then